app/controllers/concerns/signature_verification.rb

```diff
@@ -91,14 +91,21 @@ def signed_request_actor
     raise SignatureVerificationError, "Public key not found for key #{signature_params['keyId']}" if actor.nil?
 
     signature             = Base64.decode64(signature_params['signature'])
-    compare_signed_string = build_signed_string
+    compare_signed_string = build_signed_string(request_target_quirk: false)
```

Comment:
Maybe we should add a comment explaining why we need to check with and without this quirk?

Or name this parameter something more explicit?